src/main/app/src/app/informatie-objecten/informatie-object-edit/informatie-object-edit.component.ts

```diff
@@ -202,12 +203,14 @@ export class InformatieObjectEditComponent implements OnInit, OnDestroy {
       vorigeBestandsnaam = "" + titelCtrl.value;
     });
 
+    const emptyColumn = new ParagraphFormFieldBuilder().text("").build();
```

Comment:
```suggestion
    const emptyField = new ParagraphFormFieldBuilder().text("").build();
```
I'd rename this to `emptyField`, as it's added to the `this.fields` next. Minor thing.